packages/strapi-generate-new/lib/after.js

```diff
@@ -83,15 +83,33 @@ module.exports = (scope, cb) => {
   // Install default plugins and link dependencies.
   function pluginsInstallation() {
     // Define the list of default plugins.
-    const defaultPlugins = ['settings-manager', 'content-type-builder', 'content-manager', 'users-permissions', 'email'];
+    const defaultPlugins = [{
+      name: 'settings-manager',
+      core: true
+    }, {
+      name: 'content-type-builder',
+      core: true
+    }, {
+      name: 'content-manager',
+      core: true
+    }, {
+      name: 'users-permissions',
+      core: true
+    }, {
+      name: 'email',
+      core: true
+    }, {
+      name: 'analytics',
+      core: false
+    }];
 
     // Install each plugin.
     defaultPlugins.forEach(defaultPlugin => {
       try {
-        execSync(`node ${strapiBin} install ${defaultPlugin} ${scope.developerMode ? '--dev' : ''}`);
-        logger.info(`The plugin ${defaultPlugin} has been successfully installed.`);
+        execSync(`node ${strapiBin} install ${defaultPlugin.name} ${scope.developerMode && defaultPlugin.core ? '--dev' : ''}`);
```

Comment:
`defaultPlugin.core` >>> `defaultPlugins.core` (s is missing)

Reply:
Also for the `.name`

Reply:
`defaultPlugins` is an array and `defaultPlugin` the object.

Reply:
Oups, I didn't see the `forEach` loop